native-list-dir/README.md

```diff
@@ -8,49 +8,33 @@ This repository contains the code for a demo application for [GraalVM](http://gr
 
 ## Preparation
```

Comment:
Correct the link to NI documentation in the prerequisites section:
 https://www.graalvm.org/docs/reference-manual/native-image/
 to
 Native Image](https://www.graalvm.org/latest/reference-manual/native-image/